src/modules/user/routes.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import CreateUserValidation from '../../middlewares/CreateUserValidationSchema';
+import IsAuthenticatedMiddleware from '../../middlewares/IsAuthenticatedMiddleware';
+import create from './user.controller';
```

Comment:
I see you are importing `create` and not `userController` from `./user.controller`?
`./user.controller` should export functions for `create`, `delete`, `update` etc. It's ok to be explicit with the function names, to have something like `createUser`, `deleteUser` etc.